src/luarocks/init.lua

```diff
@@ -1,6 +1,12 @@
 --- LuaRocks public programmatic API, version 3.0
 local luarocks = {}
 
+local doc = require("luarocks.doc")
+luarocks = {
+   doc = doc.doc,
+   homepage = doc.homepage
+}
```

Comment:
you're assigning an empty table to `luarocks` and then overwriting it with another table.

also the idea was to keep whatever should stay at the top, and then work down refactoring the code. As such I think that the `cfg` table and the call to `init` belong in the top part because they will not be refactored